Package/gulpfile.js

```diff
@@ -124,7 +124,17 @@ const initializeSubmodulesMostRecentBabylonNative = async () => {
 
 const makeUWPProjectPlatform = async (name, arch) => {
   shelljs.mkdir('-p', `./../Modules/@babylonjs/react-native/Build/uwp_${name}`);
-  exec(`cmake -D CMAKE_SYSTEM_NAME=WindowsStore -D CMAKE_SYSTEM_VERSION=10.0 -DCMAKE_UNITY_BUILD=true -A ${arch} ./../../../react-native-windows/windows`, `./../Modules/@babylonjs/react-native/Build/uwp_${name}`);
+
+  let WindowsSDKVersionCMakeString = "";
+  const [WindowsSDKVersionFlag] = process.argv.slice(3);
+  if (WindowsSDKVersionFlag === '-WindowsSDKVersion') {
+    const [WindowsSDKVersionValue] = process.argv.slice(4);
+    if (WindowsSDKVersionValue)
+    {
+      WindowsSDKVersionCMakeString = `-DCMAKE_SYSTEM_VERSION=${WindowsSDKVersionValue}`;
+    }
+  }
```

Comment:
Would it make more sense to put this logic in tools.js (which I just noticed we have duplicated between RN versions 😬) and invoked via an npm script in package.json? This would be more like what we do for iOS, which makes it really easy to do locally in an isolated way.

Reply:
I'm not sure. tools.js is used to build the PG. and gulp.js is used to build packages. moreover, you can test this change locally with `npx gulp buildUWPPublish -WindowsSDKVersion 10.0....`